### PR TITLE
migrated tests to new workflow #373

### PR DESCRIPTION
**Pull request by cdietrich**
migrated tests to new workflow #373
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.builder.tests/src-gen/org/eclipse/xtext/builder/tests/AbstractBuilderTestLanguageRuntimeModule.java [hunk 1]
```diff
@@ -1,20 +1,58 @@
 /*
- * generated by Xtext
+ * generated by Xtext unspecified
```
Comment (kthoms) [inline org.eclipse.xtext.builder.tests/src-gen/org/eclipse/xtext/builder/tests/AbstractBuilderTestLanguageRuntimeModule.java]: change workflow

File: org.eclipse.xtext.builder.tests/src/org/eclipse/xtext/builder/tests/GenerateBuilderTestLanguages.mwe2 [hunk 2]
```diff
@@ -7,53 +7,67 @@
...
+				eclipsePluginTest = {
+					enabled = false
+				}
+				createEclipseMetaData = true
+			}
+			code = {
+				preferXtendStubs = false
+				encoding = "UTF-8"
+				lineDelimiter = "\n"
+				fileHeader = "/*\n * generated by Xtext \${version}\n */"
```
Comment (kthoms) [inline org.eclipse.xtext.builder.tests/src/org/eclipse/xtext/builder/tests/GenerateBuilderTestLanguages.mwe2]: remove version

**Comment by cdietrich** [comment]
updated

**Pull request merged**